src/helpers/helpersAPI.js

```diff
@@ -17,4 +17,36 @@ const sortLocationsByDistance = (locations = []) => {
   });
 };
 
-export { getLocationsFromResponseAPI, sortLocationsByDistance };
+const MEASUREMENTS = {
+  METER: 'mts.',
+  KILOMETERS: 'kms.',
+  MILES: 'miles.',
+};
+
+const convertDistanceMts = (distanceInMts = 0, measurement = 'miles') => {
```

Comment:
Was the issue that we were showing meters, but calling it kilometers?

Reply:
Or was the issue that we were using units that were hard to reason about (e.g., 0.1km)?

Reply:
What I noticed in prod, was it was showing that something was many kilometers away from me (like 300-1000km), but I knew it was within easy biking/walking distance. The information wasn't correct.

Reply:
Yep! 

- 'Main' problem identified was that we forgot to convert `meters` to `km` then incorrect information.
- Another issue was distance values it was less than one so small. It's a reason why now it's as meters (`toFixed(2)`).

This PR is trying to solve ^^.